src/ansys/mapdl/core/_commands/post26/set_up.py

```diff
@@ -0,0 +1,1668 @@
+           * - TF
+             - X, Y, Z, SUM
+             - Component thermal flux or vector sum.
+           * - PG
+             - X, Y, Z, SUM
+             - Component pressure gradient or vector sum.
+           * - EF
+             - X, Y, Z, SUM
+             - Component electric field or vector sum.
+           * - D
+             - X, Y, Z, SUM
+             - Component electric flux density or vector sum.
+           * - H
+             - X, Y, Z, SUM
+             - Component magnetic field intensity or vector sum.
+           * - B
+             - X, Y, Z, SUM
+             - Component magnetic flux density or vector sum.
+           * - CG
+             - X, Y, Z, SUM
+             - Component concentration gradient or vector sum.
+           * - DF
+             - X, Y, Z, SUM
+             - Component diffusion flux density or vector sum.
+           * - JC
+             - X, Y, Z, SUM
+             - Conduction current density for elements that support conduction current calculation. Components (X, Y, Z) and vector sum (SUM).
+           * - FFLX
+             - X, Y, Z
+             - Fluid-flow flux in poromechanics.
+           * - FGRA
+             - X, Y, Z
+             - Fluid pore-pressure gradient in poromechanics.
+           * - PMSV
+             - VRAT, PPRE, DSAT, RPER
+             - Void volume ratio, pore pressure, degree of saturation, and relative permeability for coupled pore-pressure-thermal elements.
+           * - NS
+             - X, Y, Z, XY, YZ, XZ
+             - `Nominal strain <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_thry/thy_mat5.html#eq4dc2eb28-41da-4d81-b0d0-8716ce41a6e1>`_ for hyperelastic material, reported in the current configuration (unaffected by :ref:`rsys` ).
+           * - MPLA
+             - DMAC, DMAX
+             - Microplane damage, macroscopic and maximum values.
+           * - MPDP
+             - TOTA, TENS, COMP, RW
+             - Microplane homogenized total, tension, and compression damages (TOTA, TENS, COMP), and split weight factor (RW).
+           * - EPFR
+             -
+             - Free strain in porous media
+           * - DAMAGE
+             - 1,2,3,MAX
+             - Damage in directions 1, 2, 3 (1, 2, 3) and the maximum damage (MAX).
+           * - GDMG
+             -
+             - Damage
+           * - IDIS
+             -
+             - Structural-thermal dissipation rate
+
+
+        .. _ANSOL_tab_2:
+
+        ANSOL - Selected Result Component Labels
+        ****************************************
+
+        .. flat-table:: Selected Result Component Labels :ref:`ansol`, ``NVAR``, ``NODE``,SRES, ``Comp``, ``Name``, ``Mat``, ``Real``, ``Ename``
+           :header-rows: 1
+
+           * - Comp
+             - Description
+           * - SVAR ``n``
+             - The ``n`` th state variable.
+           * - FLDUF0 ``n``
+             - The ``n`` th user-defined field variable.
```

Comment:
Missing third row:

<img width="695" alt="image" src="https://github.com/user-attachments/assets/4f3864e4-a184-4c96-a199-cfec3b87237c" />

Reply:
This row doesn't exist in `23.2`.

Reply:
Oohh...  I keep assuming MAPDL docs do not change 😅... sorry about that!